pyproject.toml

```diff
@@ -98,7 +103,7 @@ npm = ["jlpm"]
 
 [tool.hatch.build.hooks.jupyter-builder.editable-build-kwargs]
 path = "labextension"
-build_cmd = "install:extension"
+build_cmd = "build"
```

Comment:
I'm not 100% confident on the change from `install:extension` to `build`, but I see this be done also in [another project](https://github.com/voila-dashboards/voila/blame/0f4cc5360ff387eeaf7e647cee712b2ce08d573a/pyproject.toml#L118-L123) and Jeremy that made the that change is experienced with these matters.

---

Do you think we should have `force = true` as well as they have in that project? I think it relates to forcing rebuild of the labextension if it already exists.

https://github.com/jupyterlab/hatch-jupyter-builder/blob/12ef08143544fb954a4e5dfeb54b61f6e2ad084d/hatch_jupyter_builder/utils.py#L106-L109

But, our contributing docs mentions `jlpm watch`, which seem to trigger build of this also... Overall I'm not at all confident here, but I've seen this to work and the previous to not work at all.

I feel confident this is an incremental improvement, so I'm okay with a merge.

Reply:
I have been working on JupyterLab extensions for the past few months and I have noticed that `build` is used as a standard for editable installs in different extension sources and cookie-cutter examples. But I am happy to get feedback from JupyterLab developers.

I think we can set `force = true` without _any_ side effect. Yes, `jlpm run watch` is more convenient way to develop extensions as every change to source code will be compiled in real time.